docs/admin/networking/index.md

```diff
@@ -77,6 +77,64 @@ direct.
 In general, [port forwarded](./port-forwarding.md) web apps are faster than
 dashboard-accessed web apps.
 
+## Latency
+
+Coder measures and reports several types of latency, providing insights into the performance of your deployment. Understanding these metrics can help you diagnose issues and optimize the user experience.
+
+There are three main types of latency metrics for your Coder deployment:
+
+- Dashboard-to-server latency:
+  
+  The Coder UI measures round-trip time to the Coder server using the browser's Performance API.
+  
+  This appears in the user interface next to your username, showing how responsive the dashboard is.
+
+- Workspace connection latency:
+  
+  When users connect to workspaces, Coder measures and displays the latency between the user and:
+  - Workspace (direct P2P connection when possible)
+  - DERP relay servers (when P2P isn't possible)
+
+  This latency is visible in workspace cards and resource pages. It shows the round-trip time in milliseconds.
+
+- Database latency:
+  
+  For administrators, Coder monitors and reports database query performance in the health dashboard.
+
+### How latency is classified
+
+Latency measurements are color-coded in the dashboard:
+
+- **Green** (<150ms): Good performance.
+- **Yellow** (150-300ms): Moderate latency that might affect user experience.
+- **Red** (>300ms): High latency that will noticeably affect user experience.
+
+### View latency information
+
+- **Dashboard**: The global latency indicator appears in the top navigation bar.
+- **Workspace list**: Each workspace shows its connection latency.
+- **Health dashboard**: Administrators can view advanced metrics including database latency.
+- **CLI**: Use `coder ping <workspace>` to measure and analyze latency from the command line.
+
+### Factors that affect latency
+
+- **Geographic distance**: Physical distance between users, Coder server, and workspaces.
+- **Network connectivity**: Quality of internet connections and routing.
+- **Infrastructure**: Cloud provider regions and network optimization.
+- **P2P connectivity**: Whether direct connections can be established or relays are needed.
+
+### How to optimize latency
+
+To improve latency and user experience:
+
+- **Deploy workspace proxies**: Place [proxies](./workspace-proxies.md) in regions closer to users.
+- **Use P2P connections**: Ensure network configurations permit direct connections.
+- **Regional deployments**: Place Coder servers in regions where most users work.
```

Comment:
Careful with the pluralization here. There should only be one Coder server deployment, located somewhere close to the most users. It can have multiple replicas for scale.

Reply:
https://github.com/coder/coder/pull/16734/files#diff-f2e36a38235181ed6696007214d293eda4ee5e9c5d8ecb9665b799beaeae8e54R250-R253

```markdown
- **Deploy workspace proxies**: Place [proxies](./workspace-proxies.md) in regions closer to users, connecting back to your single Coder server deployment.
- **Use P2P connections**: Ensure network configurations permit direct connections.
- **Strategic placement**: Deploy your Coder server in a region where most users work.
- **Network configuration**: Optimize routing between users and workspaces.
```